bon-macros/src/util/ty/mod.rs

```diff
@@ -3,7 +3,7 @@ mod match_types;
 use crate::util::prelude::*;
 
 pub(crate) trait TypeExt {
-    /// Try downcasting the type to [`syn::Type::Path`]
+    /// Try down-casting the type to [`syn::Type::Path`]
```

Comment:
I believe rust's style is to write `downcast`. See how it's spelled in [std docs](https://doc.rust-lang.org/stable/std/boxed/struct.Box.html#method.downcast-2)